tests/python/base_test.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+import sys
+
+from tank_test.tank_test_base import *
+import sgtk
+from mock_host import MockHost
```

Comment:
Maybe we should repackage all of these modules under something like
```
from integration_tests import MockHost
from integration_tests import MockDescriptor
from integration_tests import TankDesktopServerFramework
```
Having a bunch of absolute includes for stuff that is not meant to be shared worries me a bit.